src/test/javascript/spec/component/team/team-participation-table.component.spec.ts

```diff
@@ -182,50 +176,54 @@ describe('TeamParticipationTableComponent', () => {
                 comp = fixture.componentInstance;
                 debugElement = fixture.debugElement;
                 teamService = TestBed.inject(TeamService);
+                router = TestBed.inject(Router);
             });
     });
 
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
     beforeEach(fakeAsync(() => {
-        const exercisesStub = sinon.stub(teamService, 'findCourseWithExercisesAndParticipationsForTeam');
-        exercisesStub.returns(of(new HttpResponse({ body: course })));
+        jest.spyOn(teamService, 'findCourseWithExercisesAndParticipationsForTeam').mockReturnValue(of(new HttpResponse({ body: course })));
         comp.course = course;
         comp.exercise = exercise4;
         comp.team = mockTeam;
+        routerStub = jest.spyOn(router, 'navigate').mockImplementation();
         comp.ngOnInit();
         tick();
     }));
 
     it('Exercises for one team are loaded correctly', fakeAsync(() => {
```

Comment:
Doesn't look like these tests need to be asynchronous. Could you please check if we can remove the fake asyncs here?

Tests that use tick() need the fakeAsync of course.

So in conclusion I'd say only "Navigate to assessment editor when opening exercise submission" should be async.

Reply:
Yes that's true. Thanks for pointing it out. I changed it.

> Changes look good to me, as Olimbia said it would be good to resolve the client style warning

Thank you very much for your feedback :-)

Reply:
Thanks for pointing this out. Indeed the fakeAsync is not necessary. I removed it.